hw/ip/otbn/README.md

```diff
@@ -22,6 +22,8 @@ An assembler, linker and disassembler for OTBN can be found in
 `hw/ip/otbn/util`. These are wrappers around a RISC-V GCC and binutils toolchain
 so one must be available (see the OpenTitan documentation on [obtaining a
 toolchain](https://docs.opentitan.org/doc/ug/install_instructions/#software-development).
```

Comment:
This is my error not yours but you could add an extra ')' here? The first closes off the link but we need another to render a closing bracket.

Reply:
I didn't notice it either! Fixed, thanks.